config/locales/en/summary.yml

```diff
@@ -89,3 +90,8 @@ en:
       hearing_date:
         question: Date of next hearing
       # END next court hearing section
+
+      # BEGIN justification for legal aid section
+      ioj_justification:
```

Comment:
So, here, in order to do proper localisation we will need to place all the IoJ types in the following format:

```yaml
loss_of_liberty:
  question: Loss of liberty
suspended_sentence:
  question: Suspended sentence
[...]
```

Is a bit verbose but it is the most flexible.